projects/iio_demo/src/main.c

```diff
@@ -201,6 +201,8 @@ int main(void)
 	iio_demo_out_init_par = (struct iio_demo_init_param) {
 		.name = demo_device_output,
 		.num_channels = 4,
+		.dev_global_attr = 1100,
+		.dev_ch_attr = 1111,
```

Comment:
Why this values?

Reply:
No particular reason, I just wanted them to be different from each other in order to see the difference in iioosciloscope.